app/templates/application.hbs

```diff
@@ -1,9 +1,9 @@
 <header>
-  <img role="heading" src="zoey-and-tom.png" width="180" height="72" alt="Ember Camp Chicago 2018 Logo">
+  <img role="heading" aria-level="1" src="embercamp-chicago-white.png" width="180" height="72" alt="Ember Camp Chicago 2018 Logo">
```

Comment:
This is gross and should probably be replaced with an actual heading

Reply:
agree- I'll PR after we merge this.